metals/src/main/scala/scala/meta/internal/decorations/SyntheticsDecorationProvider.scala

```diff
@@ -148,20 +148,21 @@ final class SyntheticsDecorationProvider(
       pcHover: Option[l.Hover],
       position: l.Position
   ): Option[l.Hover] = {
-    syntheticsAtLine
-      .find { case (range, _) =>
-        range.getEnd().getCharacter() == position.getCharacter()
-      }
-      .flatMap { case (_, synthetic) =>
-        addToHover(
-          pcHover,
-          "**Synthetics**:\n"
-            +
-              HoverMarkup(
-                synthetic
-              )
-        )
-      }
+    val interestingSynthetics = syntheticsAtLine.collect {
+      case (range, text)
+          if range.getEnd().getCharacter() == position.getCharacter() ||
+            // This happens when hover is before any first real character on line
+            range.getEnd().getCharacter() - 1 == position.getCharacter() =>
```

Comment:
I also added this case for hover at the start of the line, it seems we need to cover wider range here.

Reply:
Or actually, my bad. This is not needed.